backend/arbitre/settings.py

```diff
@@ -190,7 +190,7 @@
 # Celery Settings
 CELERY_ACCEPT_CONTENT = ["application/json"]
 CELERY_BEAT_SCHEDULER = "django_celery_beat.schedulers:DatabaseScheduler"
-CELERY_BROKER_URL = "amqp://guest:guest@localhost//"
+CELERY_BROKER_URL = "amqp://guest:guest@192.168.95.54//"
```

Comment:
⚠️ There is an hardcoded IP address
I never figured there was cases where it needed to be changed, maybe we could make it an environment variable ?

Reply:
Done